dpnp/linalg/dpnp_utils_linalg.py

```diff
@@ -569,23 +572,18 @@ def _batched_svd(
     u_matrices = [None] * batch_size
     s_matrices = [None] * batch_size
     vt_matrices = [None] * batch_size
-    ht_list_ev = [None] * batch_size * 2
     for i in range(batch_size):
         if compute_uv:
             (
                 u_matrices[i],
                 s_matrices[i],
                 vt_matrices[i],
-                ht_list_ev[2 * i],
-                ht_list_ev[2 * i + 1],
             ) = dpnp_svd(a[i], full_matrices, compute_uv=True, batch_call=True)
```

Comment:
See comment in `dpnp_svd`

```suggestion
            ) = dpnp_svd(a[i], full_matrices, compute_uv=True)
```